CHANGELOG.md

```diff
@@ -519,6 +678,78 @@ gossip.
   ([\#1558](https://github.com/cometbft/cometbft/pull/1558))
   ([\#1584](https://github.com/cometbft/cometbft/pull/1584))
 
+## v0.37.3
+
```

Comment:
Same for this hunk, why are these changes here in `v1.x`?

Reply:
will fix it as per our conversation